README.md

```diff
@@ -147,6 +147,10 @@ The [serverless framework](https://serverless.com/framework/docs/providers/aws/)
 
 - Generating Colormaps
 
+- Scale images
+
+Scaling thumbnail images utilizes a redis cache in the deployed environment. If to utilize this cache locally you'll need to install redis on the dev machine. The easiest way to do this would be by running it in a docker container such as by running `docker container run -p 1521:1521 redis:bullseye-7`. You can also use a visualizer such as `RedisInsight` to more easily inspect the cache. You will also need to set the environment variable `USE_CACHE` locally to `true` after you've started up the cache service.
```

Comment:
```suggestion
Scaling thumbnail images utilizes a redis cache in the deployed environment. To utilize this cache locally you'll need to install Redis on the dev machine. The easiest way to do this would be by running it in a docker container such as with `docker container run -p 1521:1521 redis:bullseye-7`. You can also use a visualizer such as `RedisInsight` to more easily inspect the cache. You will also need to set the environment variable `USE_CACHE` locally to `true` after you've started up the cache service.
```

Reply:
Don't include long complicated shell commands to get something to run, add a new npm script so the person reading this only has to run `npm run ...`

Reply:
Further, this should just be part of EDSC, I would imagine this starting up automatically as EDSC needs it to operate.

Reply:
Well locally we wouldn't need to cache it though. Then we also would have to bring up docker to do dev work here which since its just for thumbnail caching, is that really the default we want to have?

Reply:
If it is just a cache then I say don't make it mandatory for local dev. The fewer things that have to start up when you want to run the application the better dev experience you get. And docker is the worst

Reply:
Yea that is my preference as well docker in CMR is so finicky I I'd rather not start bringing that into EDSC